roles/python/tasks/main.yml

```
@@ -0,0 +1,9 @@
+---
+- name: Install JupyterHub
+  pip: name=jupyterhub
+- name: Install DockerSpawner
+  pip: name=dockerspawner
+- name: Install jupyter_client
+  pip: name=jupyter_client
+- name: Install jupyterhub-idle-culler
+  pip: name=jupyterhub-idle-culler
```

Maybe include these in a single block?
```suggestion
- name: Install JupyterHub and DockerSpawner
  pip:
    name:
      - jupyterhub
      - dockerspawner
      - jupyter_client
      - jupyterhub-idle-culler
```